content/en/includes/plugins/github/accounts-config-example.md

```diff
@@ -0,0 +1,29 @@
+In this example, you have `company-public` and `company-private` GitHub organizations, so you created two GitHub Apps, one for each org.
+* You installed the GitHub App for the `company-private` org with access to all repos in the org.
```

Comment:
Will revisit this example. We have changed some of the configurations and usage around the accounts

Reply:
```
github:
  plugin:
    accounts:
      - name: FirstAppRepo
        organization:  company-public
        repository: first-app-repo
        defaultBranch: master
        githubAppId: 9753
        githubAppPrivateKey: encrypted:k8s!n:spin-secrets!k:github-app-9753-privatekey
      - name: SecondAppRepo
        organization:  company-public
        repository: second-app-repo
        defaultBranch: main
        githubAppId: 9753
        githubAppPrivateKey: encrypted:k8s!n:spin-secrets!k:github-app-9753-privatekey
      - name: CompanyPrivateOrgAllRepos
        organization: company-private
        orgWideInstallation: true
        defaultBranch: main
        githubAppId: 1357
        githubAppPrivateKey: encrypted:k8s!n:spin-secrets!k:github-app-1357-privatekey
```